app/containers/ViewProfilePage/ReferralProgram.jsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+
+import H3 from 'components/H3';
+import commonMessages from 'common-messages';
+import * as routes from 'routes-config';
+import { FormattedMessage } from 'react-intl';
+
+import profileMessages from '../Profile/messages';
+import { BaseStyled } from './SettingsOfUser';
+
+const ReferralProgram = ({ className, user, writeToBuffer }) => {
+  const referralLink = `https://peeranha.io/users/${user}`;
```

Comment:
`https://peeranha.io` needs to be moved to .env file and setup per envrionment.

Reply:
Moved